fix(protocol-designer): fix hover behavior for timeline step

ncdiehl11 opened the pull request:
# Overview

I accidentally flipped the logic for handling undefined `onMouseEnter/Leave` props in `StepContainer`. This PR fixes the ternary that resulted in bad hover behavior.

## Test Plan and Hands on Testing

create or upload a protocol and verify that hovering timeline steps above any timeline error results in robot state being reflected for that step

## Changelog

- fix logic for when to pass `onMouseEnter/Leave` callback props

## Review requests

see test plan

## Risk assessment
low

TamarZanzouri committed:
feature(app): update deck configurator to have a fourth column (#18349)

jerader committed:
feat(protocol-designer): create labware quantity modal (#18440)

closes AUTH-1907

sanni-t committed:
feat(api): allow setting a fixed override to '..byVolume' properties (#18456)

Closes AUTH-1654

# Overview

Adds a method `LiquidHandlingPropertyByVolume.set_fixed_override()`
which overrides the existing 'by volume' values with the provided value

You would use this in a protocol like this-
```
water = simulated_protocol_context.define_liquid_class("water")
water_props = water.get_for(pipette_50, tiprack)

# Set flow rate of 20uL/sec for all volumes
water_props.aspirate.flow_rate_by_volume.set_for_all_volumes(20)
```

## Risk assessment

Low-None. Very small addition to functionality that doesn't change any
existing behavior.

ncdiehl11 committed:
fix(protocol-designer): fix hover behavior for timeline step

I accidentally flipped the logic for handling undefined `onMouseEnter/Leave` props in `StepContainer`. This PR fixes the ternary that resulted in bad hover behavior.

jerader reviewed (approved):
lgtm, thanks for the quick fix! don't forget to target the pd release branch

SyntaxColoring committed:
fix(app,protocol-designer): Mute <video> tags to prevent interrupting Bluetooth audio (#18459)

Closes EXEC-1588.

I don't fully understand what's going on here, but I suspect it's
something like this:

1. Navigating around the desktop app makes animations, implemented with
`<video>` tags, appear and disappear.
2. Something about that is tricking the browser, or maybe the operating
system, into thinking that you've started watching media on your
computer. To play that media's audio, it wrenches control of your
headphones away from the other connected device, and whatever that other
device was playing cuts out. Even though none of our animations actually
have audio.

Adding a `muted` attribute to every `<video>` tag in the codebase seems
to fix it.
